test/e2e/framework/pod/utils.go

```diff
@@ -127,9 +127,17 @@ const DefaultNonRootUser = 1000
 // This includes setting RunAsUser for convenience, to pass the RunAsNonRoot check.
 // Tests that require a specific user ID should override this.
 func GetRestrictedPodSecurityContext() *v1.PodSecurityContext {
+	runAsUser := pointer.Int64(DefaultNonRootUser)
+	
+	# setting RunAsUser in Windows results invalid permissions being set on projected volumes
+	# https://github.com/kubernetes/kubernetes/issues/102849
+	if framework.NodeOSDistroIs("windows") {
+		runAsUser = nil
+	}
+
 	return &v1.PodSecurityContext{
 		RunAsNonRoot:   pointer.BoolPtr(true),
-		RunAsUser:      pointer.Int64(DefaultNonRootUser),
+		RunAsUser:      runAsUser,
 		SeccompProfile: &v1.SeccompProfile{Type: v1.SeccompProfileTypeRuntimeDefault},
 	}
 }
```

Comment:
also have RunAsUser logic in MixinRestrictedPodSecurity below

Reply:
Yep, a lot of Windows clusters run linux containers on the control-plane nodes to test linux<->windows connectivity

Reply:
I've included the MixinRestrictedPodSecurity.  Will adjust as we determine if we drop or set to nil for windows